inc/templates/edit-form.tpl.php

```diff
@@ -135,3 +135,24 @@
 		<# } #>
 	</div>
 </script>
+
+<script type="text/html" id="tmpl-shortcode-ui-field-range">
+	<div class="field-block">
+		<label for="{{ data.id }}">{{ data.label }}</label>
+		<div class="field-range-container">
+			<input type="range" name="{{ data.attr }}" id="{{ data.id }}" value="{{ data.value}}" {{{ data.meta }}}  oninput="updateRangeIndicator(event)" />
+			<output class="range" for="{{ data.id }}" id="{{ data.id }}_indicator">{{ data.value}}</output>
+		</div>
+		<# if ( typeof data.description == 'string' ) { #>
+			<p class="description">{{ data.description }}</p>
+		<# } #>
+	</div>
+</script>
+
+<script>
+	function updateRangeIndicator( event ) {
```

Comment:
Why does this function have to be defined in the templates file? It seems like it makes more sense as an event handler on the `editAttributeField` view.